src/components/Articles/BlogCards.tsx

```diff
@@ -0,0 +1,126 @@
+import moment from 'moment';
+import Axios from '../../API/axios';
+import { useEffect, useState } from 'react';
+
+export interface articlesProp {
+  id?: number;
+  badge?: string;
+  imgUrl?: string;
+  title?: string;
+  writer?: string;
+  timestamp?: string;
+  videoUrl?: string;
+}
+
+const BlogCards: React.FC = () => {
```

Comment:
This is not a recommended way of creating components. it's been removed from template actually:  [Link](https://github.com/facebook/create-react-app/pull/8177)

Reply:
you can use: `const Blogcard = () => {}`